### ISSUE TITLE
Railroad diagrams are almost unreadable in prerelease

### ISSUE THREAD

**Issue by pmario** [TiddlyWiki/TiddlyWiki5#4459]
IMO this commit https://github.com/Jermolene/TiddlyWiki5/commit/152125f53b9979bfd5511973038b4358d1ef48c4 makes the railroad diagrams very low contrast with the default settings

**Comment by pmario** [comment]
see: https://tiddlywiki.com/prerelease/#Railroad%20Plugin:%5B%5BRailroad%20Plugin%5D%5D%20%5B%5BRailroad%20Diagrams%5D%5D

**Comment by kookma** [comment]
I may recommend to use black color on this background!
Now both are kind of blue!

**Comment by pmario** [comment]
[the current](https://tiddlywiki.com/#Railroad%20Plugin:%5B%5BRailroad%20Plugin%5D%5D%20%5B%5BRailroad%20Diagrams%5D%5D) version doesn't have the best settings but it is readable.

**Comment by Jermolene** [comment]
Agreed; I tried to fix it without introducing new palette colours which would still be desirable if possible.
